13:37:29.519 [main] DEBUG org.springframework.test.context.junit4.SpringJUnit4ClassRunner - SpringJUnit4ClassRunner constructor called with [class org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.527 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating CacheAwareContextLoaderDelegate from class [org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate]
13:37:29.535 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating BootstrapContext using constructor [public org.springframework.test.context.support.DefaultBootstrapContext(java.lang.Class,org.springframework.test.context.CacheAwareContextLoaderDelegate)]
13:37:29.572 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating TestContextBootstrapper for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests] from class [org.springframework.boot.test.context.SpringBootTestContextBootstrapper]
13:37:29.581 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Neither @ContextConfiguration nor @ContextHierarchy found for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests], using SpringBootContextLoader
13:37:29.585 [main] DEBUG org.springframework.test.context.support.AbstractContextLoader - Did not detect default resource location for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]: class path resource [org/springframework/cloud/kubernetes/client/discovery/KubernetesDiscoveryClientAutoConfigurationTests-context.xml] does not exist
13:37:29.586 [main] DEBUG org.springframework.test.context.support.AbstractContextLoader - Did not detect default resource location for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]: class path resource [org/springframework/cloud/kubernetes/client/discovery/KubernetesDiscoveryClientAutoConfigurationTestsContext.groovy] does not exist
13:37:29.586 [main] INFO org.springframework.test.context.support.AbstractContextLoader - Could not detect default resource locations for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]: no resource found for suffixes {-context.xml, Context.groovy}.
13:37:29.665 [main] DEBUG org.springframework.test.context.support.ActiveProfilesUtils - Could not find an 'annotation declaring class' for annotation type [org.springframework.test.context.ActiveProfiles] and class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.771 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - @TestExecutionListeners is not present for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]: using defaults.
13:37:29.772 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.event.ApplicationEventsTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener, org.springframework.test.context.event.EventPublishingTestExecutionListener]
13:37:29.782 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Skipping candidate TestExecutionListener [org.springframework.test.context.transaction.TransactionalTestExecutionListener] due to a missing dependency. Specify custom listener classes or make the default listener classes and their required dependencies available. Offending class: [org/springframework/transaction/TransactionDefinition]
13:37:29.783 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Skipping candidate TestExecutionListener [org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener] due to a missing dependency. Specify custom listener classes or make the default listener classes and their required dependencies available. Offending class: [org/springframework/transaction/interceptor/TransactionAttribute]
13:37:29.783 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@5890e879, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@6440112d, org.springframework.test.context.event.ApplicationEventsTestExecutionListener@31ea9581, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@231f98ef, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@7c137fd5, org.springframework.test.context.support.DirtiesContextTestExecutionListener@183ec003, org.springframework.test.context.event.EventPublishingTestExecutionListener@7d9d0818, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@221a3fa4, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@451001e5, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@2b40ff9c, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@3e08ff24, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@4d1c005e, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener@8462f31]
13:37:29.785 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.786 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.786 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.786 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.795 [main] DEBUG org.springframework.test.context.junit4.SpringJUnit4ClassRunner - SpringJUnit4ClassRunner constructor called with [class org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.795 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating CacheAwareContextLoaderDelegate from class [org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate]
13:37:29.796 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating BootstrapContext using constructor [public org.springframework.test.context.support.DefaultBootstrapContext(java.lang.Class,org.springframework.test.context.CacheAwareContextLoaderDelegate)]
13:37:29.796 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating TestContextBootstrapper for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests] from class [org.springframework.boot.test.context.SpringBootTestContextBootstrapper]
13:37:29.797 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Neither @ContextConfiguration nor @ContextHierarchy found for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests], using SpringBootContextLoader
13:37:29.797 [main] DEBUG org.springframework.test.context.support.AbstractContextLoader - Did not detect default resource location for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]: class path resource [org/springframework/cloud/kubernetes/client/discovery/reactive/KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests-context.xml] does not exist
13:37:29.798 [main] DEBUG org.springframework.test.context.support.AbstractContextLoader - Did not detect default resource location for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]: class path resource [org/springframework/cloud/kubernetes/client/discovery/reactive/KubernetesInformerReactiveDiscoveryClientAutoConfigurationTestsContext.groovy] does not exist
13:37:29.798 [main] INFO org.springframework.test.context.support.AbstractContextLoader - Could not detect default resource locations for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]: no resource found for suffixes {-context.xml, Context.groovy}.
13:37:29.800 [main] DEBUG org.springframework.test.context.support.ActiveProfilesUtils - Could not find an 'annotation declaring class' for annotation type [org.springframework.test.context.ActiveProfiles] and class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.817 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - @TestExecutionListeners is not present for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]: using defaults.
13:37:29.817 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.event.ApplicationEventsTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener, org.springframework.test.context.event.EventPublishingTestExecutionListener]
13:37:29.818 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Skipping candidate TestExecutionListener [org.springframework.test.context.transaction.TransactionalTestExecutionListener] due to a missing dependency. Specify custom listener classes or make the default listener classes and their required dependencies available. Offending class: [org/springframework/transaction/TransactionDefinition]
13:37:29.818 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Skipping candidate TestExecutionListener [org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener] due to a missing dependency. Specify custom listener classes or make the default listener classes and their required dependencies available. Offending class: [org/springframework/transaction/interceptor/TransactionAttribute]
13:37:29.818 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@782859e, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@23f5b5dc, org.springframework.test.context.event.ApplicationEventsTestExecutionListener@34bde49d, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@1b1cfb87, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@821330f, org.springframework.test.context.support.DirtiesContextTestExecutionListener@6f43c82, org.springframework.test.context.event.EventPublishingTestExecutionListener@5db6b9cd, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@210ab13f, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@20b2475a, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@7857fe2, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@6f15d60e, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@1be2019a, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener@29d80d2b]
13:37:29.818 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.818 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.819 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.819 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.821 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.821 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.821 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.821 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.823 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.823 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.823 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.823 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.824 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.824 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.824 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.824 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.824 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.824 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.825 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.825 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.825 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.825 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.830 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.830 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.830 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.830 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.831 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.831 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.831 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.831 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
13:37:29.833 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.833 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.833 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.833 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.837 [main] DEBUG org.springframework.test.context.support.AbstractDirtiesContextTestExecutionListener - Before test class: context [DefaultTestContext@5d908d47 testClass = KubernetesDiscoveryClientAutoConfigurationTests, testInstance = [null], testMethod = [null], testException = [null], mergedContextConfiguration = [WebMergedContextConfiguration@5efa40fe testClass = KubernetesDiscoveryClientAutoConfigurationTests, locations = '{}', classes = '{class org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests$TestConfig}', contextInitializerClasses = '[]', activeProfiles = '{}', propertySourceLocations = '{}', propertySourceProperties = '{spring.cloud.kubernetes.discovery.cacheLoadingTimeoutSeconds=5, spring.cloud.config.enabled=false, spring.cloud.kubernetes.discovery.waitCacheReady=false, org.springframework.boot.test.context.SpringBootTestContextBootstrapper=true, server.port=0}', contextCustomizers = set[org.springframework.boot.test.context.filter.ExcludeFilterContextCustomizer@662ac478, org.springframework.boot.test.json.DuplicateJsonObjectContextCustomizerFactory$DuplicateJsonObjectContextCustomizer@477b4cdf, org.springframework.boot.test.mock.mockito.MockitoContextCustomizer@0, org.springframework.boot.test.web.client.TestRestTemplateContextCustomizer@367ffa75, org.springframework.boot.test.web.reactive.server.WebTestClientContextCustomizer@1c1bbc4e, org.springframework.boot.test.autoconfigure.actuate.metrics.MetricsExportContextCustomizerFactory$DisableMetricExportContextCustomizer@2aceadd4, org.springframework.boot.test.autoconfigure.properties.PropertyMappingContextCustomizer@0, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverContextCustomizerFactory$Customizer@7bedc48a, org.springframework.boot.test.context.SpringBootTestArgs@1, org.springframework.boot.test.context.SpringBootTestWebEnvironment@7722c3c3], resourceBasePath = 'src/main/webapp', contextLoader = 'org.springframework.boot.test.context.SpringBootContextLoader', parent = [null]], attributes = map['org.springframework.test.context.web.ServletTestExecutionListener.activateListener' -> false]], class annotated with @DirtiesContext [false] with mode [null].
13:37:29.838 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.838 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
13:37:29.843 [main] DEBUG org.springframework.test.context.support.DependencyInjectionTestExecutionListener - Performing dependency injection for test context [[DefaultTestContext@5d908d47 testClass = KubernetesDiscoveryClientAutoConfigurationTests, testInstance = org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests@15d49048, testMethod = [null], testException = [null], mergedContextConfiguration = [WebMergedContextConfiguration@5efa40fe testClass = KubernetesDiscoveryClientAutoConfigurationTests, locations = '{}', classes = '{class org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests$TestConfig}', contextInitializerClasses = '[]', activeProfiles = '{}', propertySourceLocations = '{}', propertySourceProperties = '{spring.cloud.kubernetes.discovery.cacheLoadingTimeoutSeconds=5, spring.cloud.config.enabled=false, spring.cloud.kubernetes.discovery.waitCacheReady=false, org.springframework.boot.test.context.SpringBootTestContextBootstrapper=true, server.port=0}', contextCustomizers = set[org.springframework.boot.test.context.filter.ExcludeFilterContextCustomizer@662ac478, org.springframework.boot.test.json.DuplicateJsonObjectContextCustomizerFactory$DuplicateJsonObjectContextCustomizer@477b4cdf, org.springframework.boot.test.mock.mockito.MockitoContextCustomizer@0, org.springframework.boot.test.web.client.TestRestTemplateContextCustomizer@367ffa75, org.springframework.boot.test.web.reactive.server.WebTestClientContextCustomizer@1c1bbc4e, org.springframework.boot.test.autoconfigure.actuate.metrics.MetricsExportContextCustomizerFactory$DisableMetricExportContextCustomizer@2aceadd4, org.springframework.boot.test.autoconfigure.properties.PropertyMappingContextCustomizer@0, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverContextCustomizerFactory$Customizer@7bedc48a, org.springframework.boot.test.context.SpringBootTestArgs@1, org.springframework.boot.test.context.SpringBootTestWebEnvironment@7722c3c3], resourceBasePath = 'src/main/webapp', contextLoader = 'org.springframework.boot.test.context.SpringBootContextLoader', parent = [null]], attributes = map['org.springframework.test.context.web.ServletTestExecutionListener.activateListener' -> false, 'org.springframework.test.context.event.ApplicationEventsTestExecutionListener.recordApplicationEvents' -> false]]].

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v2.6.3)

2024-05-17 13:37:30.447  WARN 56414 --- [           main] ubernetesProfileEnvironmentPostProcessor : Not running inside kubernetes. Skipping 'kubernetes' profile activation.
2024-05-17 13:37:30.456  INFO 56414 --- [           main] tesDiscoveryClientAutoConfigurationTests : Starting KubernetesDiscoveryClientAutoConfigurationTests using Java 1.8.0_292 on 9168d08beda2 with PID 56414 (started by idflakies in /home/idflakies/spring-cloud/spring-cloud-kubernetes)
2024-05-17 13:37:30.457  INFO 56414 --- [           main] tesDiscoveryClientAutoConfigurationTests : No active profile set, falling back to default profiles: default
2024-05-17 13:37:31.842  INFO 56414 --- [           main] org.eclipse.jetty.util.log               : Logging initialized @3096ms to org.eclipse.jetty.util.log.Slf4jLog
2024-05-17 13:37:31.964  INFO 56414 --- [           main] org.eclipse.jetty.server.Server          : jetty-9.4.44.v20210927; built: 2021-09-27T23:02:44.612Z; git: 8da83308eeca865e495e53ef315a249d63ba9332; jvm 1.8.0_292-8u292-b10-0ubuntu1~16.04.1-b10
2024-05-17 13:37:31.982  INFO 56414 --- [           main] o.e.jetty.server.handler.ContextHandler  : Started o.e.j.s.ServletContextHandler@64f981e2{/__admin,null,AVAILABLE}
2024-05-17 13:37:31.983  INFO 56414 --- [           main] o.e.jetty.server.handler.ContextHandler  : Started o.e.j.s.ServletContextHandler@59bbb974{/,null,AVAILABLE}
2024-05-17 13:37:32.000  INFO 56414 --- [           main] o.e.jetty.server.AbstractConnector       : Started NetworkTrafficServerConnector@7c72ecc{HTTP/1.1, (http/1.1)}{0.0.0.0:33219}
2024-05-17 13:37:32.000  INFO 56414 --- [           main] org.eclipse.jetty.server.Server          : Started @3256ms
2024-05-17 13:37:32.219  INFO 56414 --- [tp2107873140-22] o.e.j.s.handler.ContextHandler.__admin   : RequestHandlerClass from context returned com.github.tomakehurst.wiremock.http.AdminRequestHandler. Normalized mapped under returned 'null'
2024-05-17 13:37:32.419  INFO 56414 --- [           main] o.s.c.a.ConfigurationClassPostProcessor  : Cannot enhance @Configuration bean definition 'kubernetesDiscoveryClientAutoConfigurationTests.TestConfig' since its singleton instance has been created too early. The typical cause is a non-static @Bean method with a BeanDefinitionRegistryPostProcessor return type: Consider declaring such methods as 'static'.
2024-05-17 13:37:32.527  INFO 56414 --- [           main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=c4d006af-a204-306e-a414-6c511b278167
2024-05-17 13:37:32.628  INFO 56414 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsAutoConfiguration' of type [io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 13:37:32.658  INFO 56414 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.cloud.commons.config.CommonsConfigAutoConfiguration' of type [org.springframework.cloud.commons.config.CommonsConfigAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 13:37:32.663  INFO 56414 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration' of type [org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 13:37:32.667  INFO 56414 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'loadBalancerClientsDefaultsMappingsProvider' of type [org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration$$Lambda$446/545392109] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 13:37:32.671  INFO 56414 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'defaultsBindHandlerAdvisor' of type [org.springframework.cloud.commons.config.DefaultsBindHandlerAdvisor] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 13:37:32.676  INFO 56414 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'kubernetes.manifests-io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsProperties' of type [io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsProperties] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 13:37:32.995  INFO 56414 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 0 (http)
2024-05-17 13:37:33.006  INFO 56414 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2024-05-17 13:37:33.007  INFO 56414 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.56]
2024-05-17 13:37:33.168  INFO 56414 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2024-05-17 13:37:33.169  INFO 56414 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 2691 ms
2024-05-17 13:37:33.396  INFO 56414 --- [oller-V1Service] i.k.client.informer.cache.Controller     : informer#Controller: ready to run resync & reflector runnable
2024-05-17 13:37:33.396  INFO 56414 --- [oller-V1Service] i.k.client.informer.cache.Controller     : informer#Controller: resync skipped due to 0 full resync period
2024-05-17 13:37:33.397  INFO 56414 --- [ler-V1Endpoints] i.k.client.informer.cache.Controller     : informer#Controller: ready to run resync & reflector runnable
2024-05-17 13:37:33.397  INFO 56414 --- [ler-V1Endpoints] i.k.client.informer.cache.Controller     : informer#Controller: resync skipped due to 0 full resync period
2024-05-17 13:37:33.399  INFO 56414 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-05-17 13:37:33.399  INFO 56414 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-05-17 13:37:33.464  INFO 56414 --- [tp2107873140-21] o.e.j.s.handler.ContextHandler.ROOT      : RequestHandlerClass from context returned com.github.tomakehurst.wiremock.http.StubRequestHandler. Normalized mapped under returned 'null'
2024-05-17 13:37:33.526 ERROR 56414 --- [tp2107873140-22] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 13:37:33.526 ERROR 56414 --- [tp2107873140-20] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 13:37:33.550 ERROR 56414 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 13:37:33.550 ERROR 56414 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 13:37:34.395  INFO 56414 --- [pool-9-thread-1] .k.c.d.KubernetesInformerDiscoveryClient : Waiting for the cache of informers to be fully loaded..
2024-05-17 13:37:34.396  INFO 56414 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Cache fully loaded (total 0 services) , discovery client is now available
2024-05-17 13:37:34.551  INFO 56414 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-05-17 13:37:34.551  INFO 56414 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-05-17 13:37:34.557 ERROR 56414 --- [tp2107873140-21] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 13:37:34.561 ERROR 56414 --- [tp2107873140-16] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 13:37:34.561 ERROR 56414 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 13:37:34.563 ERROR 56414 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 13:37:35.346  INFO 56414 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 42769 (http) with context path ''
2024-05-17 13:37:35.358  INFO 56414 --- [           main] tesDiscoveryClientAutoConfigurationTests : Started KubernetesDiscoveryClientAutoConfigurationTests in 5.49 seconds (JVM running for 6.614)
2024-05-17 13:37:35.580  INFO 56414 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-05-17 13:37:35.580  INFO 56414 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-05-17 13:37:35.589 ERROR 56414 --- [tp2107873140-21] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 13:37:35.590 ERROR 56414 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 13:37:35.592 ERROR 56414 --- [tp2107873140-20] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 13:37:35.597 ERROR 56414 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 13:37:36.056  WARN 56414 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Could not find a port named 'oops', 'https', or 'http' for service 'test-svc-1'.
2024-05-17 13:37:36.056  WARN 56414 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Make sure that either the primary-port-name label has been added to the service, or that spring.cloud.kubernetes.discovery.primary-port-name has been configured.
2024-05-17 13:37:36.056  WARN 56414 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Alternatively name the primary port 'https' or 'http'
2024-05-17 13:37:36.056  WARN 56414 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : An incorrect configuration may result in non-deterministic behaviour.
2024-05-17 13:37:36.058  WARN 56414 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Could not find a port named 'oops', 'https', or 'http' for service 'test-svc-1'.
2024-05-17 13:37:36.058  WARN 56414 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Make sure that either the primary-port-name label has been added to the service, or that spring.cloud.kubernetes.discovery.primary-port-name has been configured.
2024-05-17 13:37:36.058  WARN 56414 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Alternatively name the primary port 'https' or 'http'
2024-05-17 13:37:36.058  WARN 56414 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : An incorrect configuration may result in non-deterministic behaviour.
2024-05-17 13:37:36.069  WARN 56414 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Could not find a port named 'oops', 'https', or 'http' for service 'test-svc-1'.
2024-05-17 13:37:36.069  WARN 56414 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Make sure that either the primary-port-name label has been added to the service, or that spring.cloud.kubernetes.discovery.primary-port-name has been configured.
2024-05-17 13:37:36.069  WARN 56414 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Alternatively name the primary port 'https' or 'http'
2024-05-17 13:37:36.069  WARN 56414 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : An incorrect configuration may result in non-deterministic behaviour.

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v2.6.3)

2024-05-17 13:37:36.115  WARN 56414 --- [           main] ubernetesProfileEnvironmentPostProcessor : Not running inside kubernetes. Skipping 'kubernetes' profile activation.
2024-05-17 13:37:36.117  INFO 56414 --- [           main] iveDiscoveryClientAutoConfigurationTests : Starting KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests using Java 1.8.0_292 on 9168d08beda2 with PID 56414 (started by idflakies in /home/idflakies/spring-cloud/spring-cloud-kubernetes)
2024-05-17 13:37:36.117  INFO 56414 --- [           main] iveDiscoveryClientAutoConfigurationTests : No active profile set, falling back to default profiles: default
2024-05-17 13:37:36.314  INFO 56414 --- [           main] org.eclipse.jetty.server.Server          : jetty-9.4.44.v20210927; built: 2021-09-27T23:02:44.612Z; git: 8da83308eeca865e495e53ef315a249d63ba9332; jvm 1.8.0_292-8u292-b10-0ubuntu1~16.04.1-b10
2024-05-17 13:37:36.316  INFO 56414 --- [           main] o.e.jetty.server.handler.ContextHandler  : Started o.e.j.s.ServletContextHandler@4f581211{/__admin,null,AVAILABLE}
2024-05-17 13:37:36.316  INFO 56414 --- [           main] o.e.jetty.server.handler.ContextHandler  : Started o.e.j.s.ServletContextHandler@e0847a9{/,null,AVAILABLE}
2024-05-17 13:37:36.318  INFO 56414 --- [           main] o.e.jetty.server.AbstractConnector       : Started NetworkTrafficServerConnector@36871e98{HTTP/1.1, (http/1.1)}{0.0.0.0:41949}
2024-05-17 13:37:36.318  INFO 56414 --- [           main] org.eclipse.jetty.server.Server          : Started @7574ms
2024-05-17 13:37:36.322  INFO 56414 --- [tp1517972788-50] o.e.j.s.handler.ContextHandler.__admin   : RequestHandlerClass from context returned com.github.tomakehurst.wiremock.http.AdminRequestHandler. Normalized mapped under returned 'null'
2024-05-17 13:37:36.369  INFO 56414 --- [           main] o.s.c.a.ConfigurationClassPostProcessor  : Cannot enhance @Configuration bean definition 'kubernetesInformerReactiveDiscoveryClientAutoConfigurationTests.TestConfig' since its singleton instance has been created too early. The typical cause is a non-static @Bean method with a BeanDefinitionRegistryPostProcessor return type: Consider declaring such methods as 'static'.
2024-05-17 13:37:36.373  INFO 56414 --- [           main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=d7232dc4-7a46-3e4e-9978-4dacfc5f6386
2024-05-17 13:37:36.395  INFO 56414 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsAutoConfiguration' of type [io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 13:37:36.406  INFO 56414 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.cloud.commons.config.CommonsConfigAutoConfiguration' of type [org.springframework.cloud.commons.config.CommonsConfigAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 13:37:36.408  INFO 56414 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration' of type [org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 13:37:36.410  INFO 56414 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'loadBalancerClientsDefaultsMappingsProvider' of type [org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration$$Lambda$446/545392109] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 13:37:36.412  INFO 56414 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'defaultsBindHandlerAdvisor' of type [org.springframework.cloud.commons.config.DefaultsBindHandlerAdvisor] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 13:37:36.414  INFO 56414 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'kubernetes.manifests-io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsProperties' of type [io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsProperties] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 13:37:36.480  INFO 56414 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 0 (http)
2024-05-17 13:37:36.481  INFO 56414 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2024-05-17 13:37:36.481  INFO 56414 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.56]
2024-05-17 13:37:36.591  INFO 56414 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-05-17 13:37:36.598  INFO 56414 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-05-17 13:37:36.599 ERROR 56414 --- [tp2107873140-21] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 13:37:36.601 ERROR 56414 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 13:37:36.608 ERROR 56414 --- [tp2107873140-15] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 13:37:36.611 ERROR 56414 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 13:37:36.876  INFO 56414 --- [oller-V1Service] i.k.client.informer.cache.Controller     : informer#Controller: ready to run resync & reflector runnable
2024-05-17 13:37:36.876  INFO 56414 --- [ler-V1Endpoints] i.k.client.informer.cache.Controller     : informer#Controller: ready to run resync & reflector runnable
2024-05-17 13:37:36.876  INFO 56414 --- [oller-V1Service] i.k.client.informer.cache.Controller     : informer#Controller: resync skipped due to 0 full resync period
2024-05-17 13:37:36.876  INFO 56414 --- [ler-V1Endpoints] i.k.client.informer.cache.Controller     : informer#Controller: resync skipped due to 0 full resync period
2024-05-17 13:37:36.877  INFO 56414 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-05-17 13:37:36.877  INFO 56414 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-05-17 13:37:36.878  INFO 56414 --- [tp1517972788-43] o.e.j.s.handler.ContextHandler.ROOT      : RequestHandlerClass from context returned com.github.tomakehurst.wiremock.http.StubRequestHandler. Normalized mapped under returned 'null'
2024-05-17 13:37:36.891 ERROR 56414 --- [tp1517972788-48] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 13:37:36.894 ERROR 56414 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 13:37:36.901 ERROR 56414 --- [tp1517972788-43] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 13:37:36.903 ERROR 56414 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 13:37:37.602  INFO 56414 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-05-17 13:37:37.607 ERROR 56414 --- [tp2107873140-21] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 13:37:37.609 ERROR 56414 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 13:37:37.612  INFO 56414 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-05-17 13:37:37.618 ERROR 56414 --- [tp2107873140-15] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 13:37:37.620 ERROR 56414 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 13:37:37.876  INFO 56414 --- [ool-27-thread-1] .k.c.d.KubernetesInformerDiscoveryClient : Waiting for the cache of informers to be fully loaded..
2024-05-17 13:37:37.876  INFO 56414 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Cache fully loaded (total 0 services) , discovery client is now available
2024-05-17 13:37:37.895  INFO 56414 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-05-17 13:37:37.900 ERROR 56414 --- [tp1517972788-49] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 13:37:37.902 ERROR 56414 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 13:37:37.904  INFO 56414 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-05-17 13:37:37.912 ERROR 56414 --- [tp1517972788-43] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 13:37:37.914 ERROR 56414 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 13:37:38.143  INFO 56414 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 38503 (http) with context path ''
2024-05-17 13:37:38.147  INFO 56414 --- [           main] iveDiscoveryClientAutoConfigurationTests : Started KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests in 2.064 seconds (JVM running for 9.403)
